endpoints/API_Player.py

```diff
@@ -2,235 +2,201 @@
-        self.write(msg='Fetched car details of :%s %s' % (car_name, response))
+        login_data={'username': user,'password':password}
+        return login_data
 
-        return result_flag
 
-    
-    def add_car(self, car_details, auth_details=None):
-        "adds a new car"
-        data = car_details
-        headers = self.set_header_details(auth_details)
-        json_response = self.api_obj.add_car(data=data,
-                headers=headers)
-        result_flag = True if json_response['response']['successful'] == True else False
-  
-        return result_flag
+    def login_app(self,login_data):
+        "login to app"
+        response = self.api_obj.login_app_is(data=login_data)
+        result_flag = True if response['response'] == 200 else False
 
-    
-    def register_car(self, car_name, brand, auth_details=None):
-        "register car"
-        url_params = {'car_name': car_name, 'brand': brand}
-        url_params_encoded = urllib.parse.urlencode(url_params)
-        customer_details = conf.customer_details
-        data = customer_details
-        headers = self.set_header_details(auth_details)
-        json_response = self.api_obj.register_car(url_params=url_params_encoded,
-                                                  json=data,
-                                                  headers=headers)
-        response = (json_response['response'])
-        result_flag = True if response['registered_car']['successful'] == True else False        
-        
         return result_flag
-    
-    
-    def update_car(self, car_details, car_name='figo', auth_details=None):
-        "updates a car"
-        data = {'name': car_details['name'],
-                'brand': car_details['brand'],
-                'price_range': car_details['price_range'],
-                'car_type': car_details['car_type']}
-
-        headers = self.set_header_details(auth_details)
-        json_response = self.api_obj.update_car(car_name,
-                                                json=data,
-                                                headers=headers)
-        json_response = json_response['response']
-        result_flag = True if json_response['response']['successful'] == True else False
+
+
+    def get_jobs(self):
+        "get available jobs"
+        response = self.api_obj.get_jobs_is()
+        result_flag = True if response['response'] == 200 else False
+        ses = response['response_content']
+        soup = BeautifulSoup(ses)
+        My_table = soup.find('table',{'class':'table table-striped'})
+        table_rows = My_table.find_all('tr')
+        list_1=[]
+        for tr in table_rows:
+            td = tr.find_all('td')
+            row = [i.text for i in td]
+            list_1.append(row)
+        list_1.pop(0)
+        list_2 = []
+        for val in list_1:
+            list_2.append(val[1])
+        list_2 = [x.replace("\n","") for x in list_2]
+        self.new_job_id = row[0]
+
```

Comment:
These previous lines are mentioned in all method names shall we move to one method and shall we call it from here